Document/0x06d-Testing-Data-Storage.md

```diff
@@ -802,112 +802,49 @@ In summary, when performing static analysis for sensitive data exposed via memor
 
 #### Dynamic Analysis
 
-Several approaches and tools are available for dumping an iOS app's memory.
+Several approaches and tools are available for testing the memory of an iOS app.
```

Comment:
so are we testing the memory? or are we searching it? or testing it for the presence of sensitive information? or... dumping alone is a bit too little indeed, but what are we testing it for :) ?

Reply:
done